### PR TITLE
Recover w/link: Add UI for error scenario

### PR DESCRIPTION
**Pull request by Patrick1904**

### CODE REVIEW

File: src/components/accounts/RecoverWithLink.js [hunk 1]
```diff
@@ -92,17 +111,32 @@ class RecoverWithLink extends Component {
...
             seedPhrase: this.props.seedPhrase,
-            successSnackbar: false
+            successSnackbar: false,
+            successView: true
         };
     }
 
+    componentWillMount = () => {
+        if (!this.isLegit) {
+            this.setState({ successView: false });
```
Comment (vgrichina) [inline src/components/accounts/RecoverWithLink.js]: if you want to create derived property, why not just create it like this?

```
get successView() {
      return this.isLegit;
}
```

trying to track it in state is far more error prone

Comment (Patrick1904) [reply src/components/accounts/RecoverWithLink.js]: Very nice

**Comment by vgrichina** [comment]
@Patrick1904 please move it to our repo, otherwise it doesn't get deployed to Render (and so hard to test for me and @kcole16).

**Comment by Patrick1904** [comment]
This PR is obsolete now. Closing.

**Pull request closed without merging**